hypothesis-python/tests/cover/test_targeting.py

```diff
@@ -78,9 +78,8 @@ def everything_except(type_):
 @example(float("nan"), "")
 @example(float("inf"), "")
 @example(float("-inf"), "")
-@example("1", "Non-float observations are invalid")
 @example(0.0, ["a list of strings is not a valid label"])
-@given(observation=everything_except(float), label=everything_except(str))
+@given(observation=everything_except(float or int), label=everything_except(str))
```

Comment:
```suggestion
@given(observation=everything_except((float, int)), label=everything_except(str))
```

Turns out that `float or int` evaluates to `float`, becuase `bool(float) is True`!